### PR TITLE
Add a final question to know if the issue submitter will provide a PR

### PR DESCRIPTION
**Pull request by bmarty**
Small improvement on the issue template.

Inspired from https://github.com/Kotlin/dokka/blob/master/.github/ISSUE_TEMPLATE/bug.md

@kittykat WDYT?

### CODE REVIEW

**Review by onurays — APPROVED** [review]
LGTM! (This seems like a very good idea to me)

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -73,3 +73,14 @@ body:
...
+    id: pr
+    attributes:
+      label: Are you willing to provide a PR?
+      description:  |
+        Providing a PR can drastically speed up the process of fixing this bug. Don't worry, it's still OK to answer 'No' :).
+      options:
+        - 'Yes'
+        - 'No'
+    validations:
+      required: true
```
Comment (kittykat) [inline .github/ISSUE_TEMPLATE/bug.yml:86]: What is your logic for making this a required question?

Comment (bmarty) [reply .github/ISSUE_TEMPLATE/bug.yml:86]: Was just copy pasting the value from the question above `Will you send logs?`. To me this is the same level of requirement, and I guess we want the issue submitter to answer every questions.

But happy to set this one to `false` if you prefer.

**Comment by bmarty** [comment]
Let's merge and iterate if we want to.

**Pull request merged**